<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Segédprogram madárgyűrűzéshez</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRingCode">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEditCaptureDate">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2024</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxSpecies">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>421</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEditCaptureLocation">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gyűrű kódja</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Madár faja</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Befogás ideje</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Befogás helye</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>160</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxAge">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>P</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>F</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1y</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2y</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3y</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3+</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ivar</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxSex">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>H</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>T</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>180</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Befogás típusa</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxCaptureType">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>210</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>F</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>R</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>H</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>E</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Státusz</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxStatus">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>N</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>K</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>L</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>P</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>R</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>T</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>W</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>U</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Manipuláció</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxManipulation">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>N</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>H</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>K</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>P</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>R</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>F</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>T</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>U</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Save">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>290</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Madár adatainak felvétele</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchLineEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="birdDataTable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>731</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>Gyűrű kód</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Faj</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Befogás ideje</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Befogás helye</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Befogás típusa</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Kor</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ivar</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Státusz</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Manipuláció</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="Delete">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>300</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Adatsor törlése</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>787</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menü</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="actionReset"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImport">
   <property name="text">
    <string>Adatok betöltése fájlból...</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Adatok betöltése fájlból&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans MT</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Adatok kimentése...</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Adatok visszaállítása...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
